cloud: add aws s3 and gcs config (#9735)
pingcap/docs: tidb-cloud/config-s3-and-gcs-access.md

---
title: Configure Amazon S3 access and GCS access
summary: Learn how to configure Amazon Simple Storage Service (Amazon S3) access and Google Cloud Storage (GCS) access.
---

# Configure Amazon S3 access and GCS access

When you import or migrate from Amazon S3 or GCS to TiDB Cloud, you need to configure access to the Amazon S3 or GCS buckets. This document describes how to do this.

> 

Author: ```suggestion

- [Configure Amazon S3 access](#configure-amazon-s3-access)
- [Configure Google Cloud Storage access](#configure-google-cloud-storage-access)

```

---

> title: Configure Amazon S3 access and GCS access

Reviewer: ```suggestion
title: Configure Amazon S3 Access and GCS Access
```

summary: Learn how to configure Amazon Simple Storage Service (Amazon S3) access and Google Cloud Storage (GCS) access.
---

> # Configure Amazon S3 access and GCS access

Reviewer: ```suggestion
# Configure Amazon S3 Access and GCS Access
```

> When you import or migrate from Amazon S3 or GCS to TiDB Cloud, you need to configure access to the Amazon S3 or GCS buckets. This document describes how to do this.

Reviewer: ```suggestion
If your source data is stored in Amazon S3 or GCS buckets, before importing or migrating the data to TiDB Cloud, you need to configure cross-account access to the buckets. This document describes how to do this.
```
